src/stylesheets/common/_buttons.scss

```diff
@@ -22,12 +22,17 @@ input[type="search"] + .btn {
 
 // Dark purple button
 .btn-mapzen {
-  background-color: nth($mz-darkpurples, 1);
-  color: #ffffff;
+  @extend .btn-primary;
+}
+
+.btn-primary {
+  background-color: $mz-darkpurple-1;
```

Comment:
I snuck those in here: https://github.com/mapzen/styleguide/pull/423/files

(You can see the addition of RGB, hex, and variable names live: https://mapzen.com/common/styleguide/design-elements.html#colors)

The `nth($mz-color, X)` format was not documented in the color section, so I opted to create the variables, which I feel are easier to understand. Referring to a Sass list seems unnatural and unwieldy when a variable is simpler to grok and to type. The `$mz-color-X` variants are meant to be a convenience which requires the least amount of extra knowledge and understanding of Sass to use. You can still use the `nth()` method if that's preferable.